x-pack/plugins/cloud_security_posture/public/pages/vulnerabilities/vulnerabilities_finding_flyout/vulnerability_overview_tab.tsx

```diff
@@ -161,7 +161,7 @@ const VulnerabilityOverviewTiles = ({ vulnerabilityRecord }: VulnerabilityTabPro
 
   return (
     <EuiFlexGroup data-test-subj={OVERVIEW_TAB_VULNERABILITY_FLYOUT}>
```

Comment:
The issue arises because JavaScript treats 0 as a falsy value, but in the case of your condition, the vulnerability?.score?.base && vulnerability.score.base !== 0 is evaluated separately. Since vulnerability?.score?.base returns 0 (falsy), the component still renders because it’s not stopping evaluation correctly.

Reply:
why `base` is 0 in the first place? is `wiz.vulnerability.score` 0 when it's missing?

Reply:
looks like 0 is a valid value. we just dont handle it and didnt expect it as such. ive talked with ido, for now i guess just ignore 0 and we will take this with @smriti-elastic